JustSaying/JustSaying.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard1.6;net451</TargetFrameworks>
+    <LangVersion>7.1</LangVersion>
```

Comment:
Could just define these in `Directory.Build.props` instead?

Reply:
👍